### DISCUSSION TITLE
银保监会回应「停贷」事件，称「引导金融机构市场化参与风险处置」，会带来哪些影响？

### DISCUSSION THREAD

**Discussion by WeNeedHome** [WeNeedHome/SummaryOfLoanSuspension#450] [General]
![48d95512ece239ccc726cc7d4e51412](https://user-images.githubusercontent.com/109166931/179145289-1b9ef3ff-e883-4205-a8fe-30ef564b9a4f.png)

https://www.zhihu.com/question/543313326/answer/2574855307

**Comment by DarkForestBlack** [comment]
银保监太会说话了。说了一大堆，却没有啥意义，滴水不漏。看不懂的人大有人在，还得有人解读，至于这个解读的对不对，就不是银保监的事情了。
目前看到的网上的一个比较合理的解读：
1先隐晦的承认错误。
2不正面回应断贷，淡化影响。
3积极配合争取交付。解决一部分打压一部分
4最后皆大欢喜

**Comment by tiancaipipi110** [comment]
![R](https://user-images.githubusercontent.com/35275506/179162371-702dc7af-3b0d-4db7-b0fb-5882d5d81530.jpg)

**Comment by zxcs001** [comment]
这个答案已经没有了，全网要和谐这个话题了
